orca-bakery/src/test/groovy/com/netflix/spinnaker/orca/bakery/tasks/CreateBakeTaskSpec.groovy

```diff
@@ -292,29 +292,6 @@ class CreateBakeTaskSpec extends Specification {
   }
 
   @Unroll
-  def "fails if pipeline trigger or context includes artifacts but no artifact for the bake package"() {
```

Comment:
Can this test case be added back now that the `allowMissingPackageInstallation` property has been introduced, and defaults to `false`?